test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -48,7 +61,7 @@ class database_operation {
      * defined by the configuration.
      *      - Store in memory the created collections.
      */
-    virtual void
```

Comment:
Doesn't look like we're overriding these nor do we intend to.

Reply:
Doesn't example.cxx override it?

Reply:
Probably, I'll check when I get in. Are they marked as override in your PR? If not, I'll do it here.

Reply:
Coming in my PR

Reply:
80388e3